universe/interface.go

```diff
@@ -122,13 +122,16 @@ type LeafKey struct {
 	// TODO(roasbeef): add asset type too?
 }
 
-// UniverseKey is the key for a universe.
-func (b LeafKey) UniverseKey() [32]byte {
-	// key = sha256(mintingOutpoint || scriptKey)
+// Hash returns the hash bytes of a LeafKey. The returned value is used as a key
+// within a universe's MS-SMT tree.
+func (b LeafKey) Hash() [32]byte {
```

Comment:
I don't think this rename is warranted: it isn't the hash of the value, it's the key to use when inserting into a universe server.

Reply:
I've removed this rename.